### PR TITLE
script: add some init script

### PR DESCRIPTION
**Pull request by slievrly**
Signed-off-by: slievrly <slievrly@163.com>

### Ⅰ. Describe what this PR did

script: add some init script

### CODE REVIEW

File: script/client/spring/application.properties [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+seata.tx-service-group=my_test_tx_group
+seata.client.rm-async-commit-buffer-limit=1000
+seata.client.rm-report-retry-count=5
+seata.client.rm-table-meta-check-enable=true
+seata.client.tm-commit-retry-count=5
+seata.client.tm-rollback-retry-count=5
+seata.client.rm.lock.lock-retry-interval=10
+seata.client.rm.lock.lock-retry-times=30
+seata.client.rm.lock.lock-retry-policy-branch-rollback-on-conflict=true
+seata.client.support.spring.datasource-autoproxy=true
```
Comment (zjinlei) [inline script/client/spring/application.properties]: default false

File: script/client/spring/application.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    tm-commit-retry-count: 5
+    tm-rollback-retry-count: 5
+    rm:
+      lock:
+        lock-retry-interval: 10
+        lock-retry-times: 30
+        lock-retry-policy-branch-rollback-on-conflict: true
+    support:
+      spring:
+        datasource-autoproxy: true
```
Comment (zjinlei) [inline script/client/spring/application.yml]: default false

File: script/client/spring/application.properties (hunk 1 above)
Comment (xingfudeshi) [reply script/client/spring/application.properties]: For file.conf,the default value is false.But for seata-spring-boot-starter,the value is true as default.

File: script/client/spring/application.yml (hunk 2 above)
Comment (slievrly) [reply script/client/spring/application.yml]: springboot set default value true

File: script/client/spring/application.yml [hunk 3]
```diff
@@ -0,0 +1,95 @@
+seata:
+  enabled: true
+  application-id: test
```
[the remark this answers is not in the document]
Comment (slievrly) [reply script/client/spring/application.yml]: io.seata.spring.boot.autoconfigure.properties.applicationId

**Comment by slievrly** [comment]
@xingfudeshi `seata.config.file.name` and `seata.registry.file.name=file.conf` is necessary?

**Pull request merged**